e2e-chainsaw/linodecluster-controller/minimal-linodecluster/chainsaw-test.yaml

```diff
@@ -0,0 +1,75 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: minimal-linodecluster
+spec:
+  steps:
+  - name: step-00
+    try:
+    - assert:
+        file: 00-assert.yaml
+  - name: step-01
+    try:
+    - apply:
+        file: 01-create-cluster.yaml
+  - name: step-02
+    try:
+    - apply:
+        file: 02-create-linodecluster.yaml
+    - assert:
+        file: 02-assert.yaml
+  - name: step-03
+    try:
+    - script:
+        env:
+        - name: TARGET_API
+          value: api.linode.com
+        - name: TARGET_API_VERSION
+          value: v4beta
+        - name: URI
+          value: nodebalancers
+        - name: FILTER
+          value: '{"label":"linodecluster-test-api-server"}'
+        content: |
+          set -e
+          curl -s \
+            -H "Authorization: Bearer $LINODE_TOKEN" \
+            -H "X-Filter: $FILTER" \
+            -H "Content-Type: application/json" \
+            "https://$TARGET_API/$TARGET_API_VERSION/$URI"
+        # TODO: check results
```

Comment:
@eljohnson92 i replaced the makefile target with an inline script

Reply:
If the payload returned is json it's easy to write a robust assertion

Reply:
ah, yes, I like the way that is laid out. it is a json response and I would much prefer checking values within the json, is that something we can do natively with Chainsaw or would we have to use JQ to handle checking fields/values?

Reply:
@eljohnson92 i implemented the robust checks 🤞

Reply:
@eljohnson92 i implemented the check, it's natively supported